src/js/view/controls/components/volumetooltipicon.ts

```diff
@@ -50,6 +50,15 @@ export default class VolumeTooltipIcon extends TooltipIcon {
         horizontalContainer.appendChild(this.horizontalSlider.element());
         this.addContent(this.verticalSlider.element());
 
+        const hasHorizontalSlider = _model.get('horizontalVolumeSlider') || _model.get('audioMode');
+
+        if (hasHorizontalSlider) {
+            this.horizontalContainer.style.display = 'flex';
+            this.tooltip.style.display = 'none';
+        }
+
+        this.setHorizontalSliderTabIndex(null, hasHorizontalSlider);
```

Comment:
Maybe you should do this below in what used to be `onAudioMode` since    `horizontalVolumeSlider` proceeds `audioMode`.